### PR TITLE
Add first draft of client usage docs

### PR DESCRIPTION
**Pull request by jhiemstrawisc**

### CODE REVIEW

**Comment by joereuss12** [comment]
It looks very good to me, detailed with many examples. All good from my side but we can see what Emma thinks too before we merge.

**Comment by turetske** [comment]
I highly recommend checking your final version via Cannon's docs repository if you have not already done so as I found a few syntax bugs in my own that prevented the page from being rendered when I did.

File: docs/pages/client-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+pelican object copy -f <Federation URL> </federation/path/to/file> </local/path/to/file> -t </path/to/token/file>
+```
+
+For example, if we assume the following token grants read access to the `/ospool/PROTECTED` namespace:
+```console
+eyJhbGciOiJSUzI1NiIsImtpZCI6ImtleS1yczI1NiIsInR5cCI6IkpXVCJ9.eyJ2ZXIiOiJzY2l0b2tlbjoyLjAiLCJhdWQiOiJodHRwczovL2RlbW8uc2NpdG9rZW5zLm9yZyIsImlzcyI6Imh0dHBzOi8vZGVtby5zY2l0b2tlbnMub3JnIiwiZXhwIjoxNjk3NDgzNTU5LCJpYXQiOjE2OTc0ODI5NTksIm5iZiI6MTY5NzQ4Mjk1OSwianRpIjoiOGIzNjQ5MTUtMjM4MC00MzM2LWI1OTktN2NmYzhiNGJmNTk3In0.hCf8oi3BRoWnUrBxSKST8p8czSChetMFID4FRXiQQ6RnwhWFZD3grZ2dvdYIYYDuW-1iATN9OujHBbO8TOxTnjJd7acE7la5rZscQwY_DAr_6rLKRTSU_Tpgg8uBMQB-U45nGWJVuYS6RZ3JZ2vE5lTtvPjZjExkJOkfvVp9Kzq445UGlK4dNkvTS3SYd9QYiZPkjA_Z-u57DesOOhsgrLSXyrRCtxBD8mRe5MiRtVAFHxIXS_ZQ7B2XlmNPiR6PBb9r38qHUlYe9y824hmBW-VzR2xiJd5wLWFZOv2Ec-q2NCAqDQfGYl4UsWKinW-35OGEULQWAQgHwxKJMSEH8A
+```
+
+Then the token can be used to get the object `/ospool/PROTECTED/justin.hiemstra/auth-test.txt` by saving the token in a file called `my-token` and running
+
```
Comment (turetske) [inline docs/pages/client-usage.mdx:91]: Do we want direct references to ourselves? Specifically the 'justin.hiemstra' folder? Maybe make it more generic?

**Review by turetske — CHANGES_REQUESTED** [review]
LGTM with minor changes

**Comment by jhiemstrawisc** [comment]
One other quick comment -- it looks like each tab in the rendered index just reads the file name. Can you name the document something like `serving_an_origin.mdx`?

**Review by joereuss12 — APPROVED** [review]
LGTM

**Review by turetske — APPROVED** [review]
LGTM!

**Pull request merged**